ecpy_hqc_legacy/instruments/drivers/visa/tektro_awg.py

```diff
@@ -425,6 +447,15 @@ def reopen_connection(self):
             except VisaIOError:
                 break
 
+    def clear_output_buffer(self):
+        self.timeout = 100
+        while True:
+            try:
+                self.read()
+            except VisaIOError:
+                self.timeout = 2000
```

Comment:
Here you should restore the previous value not set an arbitrary one.

Reply:
ok